doc/tapyrus/schnorr_signature.md

```diff
@@ -10,14 +10,30 @@ Signing algorithm:
 * The message m: a 32-byte array
 
 To sign m for public key pubkey(sk):   
-1. Let d' = int(sk).  
-1. Fail if d' = 0 or d' >= n.  
-1. Let P = d'G.  
-1. Let k be the nonce generated using rfc6979.  
-1. Let R = k'G.   
-1. Let k = k' if jacobi(y(R)) = 1, otherwise let k = n - k'.   
-1. Let e = int(sha256(bytes(R) || bytes(P) || m)) mod n.   
-1. The signature is bytes(x(R)) || bytes((k + ed) mod n).   
+1. Let d' = int(sk).
+1. Fail if d' = 0 or d' >= n.
```

Comment:
Here, `n` comes out for the first time, so it is necessary to explain what `n` is.